ecosystem/sep-0008.md

```diff
@@ -226,6 +226,8 @@ Name | Type | Description
 
 This response means that the user must complete an action before this transaction can be approved. The approval service will provide a URL that facilitates the action. Upon completion, the user will resubmit the transaction.
 
+The action the user must complete is interactive, and is intended to be displayed in a browser. The client may be able to skip the need to display the URL in the browser if it can supply the requested details.
```

Comment:
```suggestion
The default action the user must complete is interactive, and is intended to be displayed in a browser. The client may be able to skip the need to display the URL in the browser if it can supply the requested details.
```
Iirc it's only interactive if the `action_method` is `GET`, which is the default method if undefined.

Reply:
It's also interactive in the POST case if the data included is not sufficient.

Reply:
That's right, but it won't be if the data included is sufficient. Just wanted to point this out.

Or we can do this or something similar
```suggestion
The action the user must complete is interactive unless the client is able to provide sufficient data with a POST request, and is intended to be displayed in a browser. The client may be able to skip the need to display the URL in the browser if it can supply the requested details.
```

Reply:
I took your edit and extended it in ec706df. Thanks!